test/utils/hex.hpp

```diff
@@ -28,3 +28,9 @@ inline std::string hex(bytes_view data)
     return hex(data.data(), data.size());
 }
 }  // namespace fizzy
+
+/// Operator for "" literals, e.g. "0a0b0c0d"_hex.
+inline fizzy::bytes operator""_hex(const char* literal, size_t /*length*/)
```

Comment:
Can't it be defined inside `namespace fizzy` ?

Reply:
Can be, but then `using namespace fizzy` is required.

Reply:
I think it's good, it's similar to like standard custom literals are in `namespace std` (and even in the nested inline namespaces). And we do `using namespace fizzy` in tests anyway.